.env.j2

```diff
@@ -8,7 +8,14 @@ FLASK_DEBUG=true
 # Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
 SECRET_KEY=secret-key
 
+# Required to use HTTPS URLs
+PREFERRED_URL_SCHEME=https
+
 # Required to use MongoDB
 MONGO_CONNECTION_STRING={{ MONGO_CONNECTION_STRING }}
 MONGO_DB_NAME={{ MONGO_DB_NAME }}
-MONBGO_DB_COLLECTION={{ MONBGO_DB_COLLECTION }}
\ No newline at end of file
+MONGO_DB_COLLECTION={{ MONGO_DB_COLLECTION }}
```

Comment:
This is not set correctly on the App Service atm:
![image](https://github.com/user-attachments/assets/5f3b6dc2-7c56-409d-8ddc-73102b574b81)
![image](https://github.com/user-attachments/assets/467ddf3f-aee9-44ac-ad8e-8800ebee1ff3)

Reply:
I've updated the env variable on the app service and it now appears to be working :)

Reply:
Interesting, I wonder how it was working before 😂